modules/web/js/ballerina/components/block-statement-decorator.jsx

```diff
@@ -26,16 +26,20 @@ import ExpressionEditor from 'expression_editor_utils';
 
 class BlockStatementDecorator extends React.Component {
 
-	render() {
-		const { bBox, title, dropTarget , expression } = this.props;
+    render() {
+        const {bBox, title, dropTarget, expression} = this.props;
         let title_h = blockStatement.heading.height;
         let title_w = blockStatement.heading.width;
 
+        if (this.props.titleViewState) {
```

Comment:
better to put a comment explaining why we need titleview state. Or we can simply pass one prop mentioning title width since we don't use other attr ie h, x, y.